pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
@gfleeming . Thank you for your contribution. I think there is a typo here. Probably it should be:
```
def subtract(x, y):
     return x - y
```